resources/update-ljm.sh

```diff
@@ -2,15 +2,49 @@
 
 set -e -u
 
+if [[ ! -z $(git status -s) ]]; then
```

Comment:
I think this should be: git status -s --untracked-files=no
I always have untracked files in the repo :)